ckanext/datasetapproval/validation.py

```diff
@@ -19,12 +19,12 @@ def state_validator(key, data, errors, context):
         if org.get('id') == office_id:
             if org.get('capacity') == 'admin':
                 # If no state provided and user is an admin, default to active
-                state = state or 'active'
+                state = state 
```

Comment:
@sagargg do we need to assign state = state here, given that state is already going to be = state?